docroot/profiles/custom/cgov_site/modules/custom/cgov_core/config/install/metatag.metatag_defaults.node.yml

```diff
@@ -5,13 +5,14 @@ id: node
 label: Content
 tags:
   canonical_url: '[node:url]'
+  cgdp_template: default
```

Comment:
@KateMashkinaNIH this is one of those places where we need to be 100% sure that default shows up on all nodes that are not one one of the types where this comes from a field. This includes Spanish nodes too.